src/main.js

```diff
@@ -3,6 +3,18 @@
 const electron = require('electron');
 const app = electron.app; // Module to control application life.
 
+var cmd = process.argv[1];
+if (cmd === '--squirrel-uninstall') {
+  var AutoLaunch = require('auto-launch');
+  var appLauncher = new AutoLaunch({
+    name: 'Mattermost'
+  });
+  appLauncher.isEnabled().then(function(enabled) {
+    if (enabled)
+      appLauncher.disable();
+  });
+}
+
 if (require('electron-squirrel-startup')) app.quit();
```

Comment:
I don't think it will change anything or matter, looking at https://github.com/mongodb-js/electron-squirrel-startup/blob/master/index.js
I'm not sure why they are saying that there should be an if around it in the docs